src/simulator/src/data.ts

```diff
@@ -0,0 +1,83 @@
+import { fullView } from './ux'
+import { createSubCircuitPrompt } from './subcircuit'
+import save from './data/save'
+import load from './data/load'
+import createSaveAsImgPrompt from './data/saveImage'
+import {
+    clearProject,
+    newProject,
+    saveOffline,
+    openOffline,
+    recoverProject,
+} from './data/project'
+import { createNewCircuitScope } from './circuit'
+import { createCombinationalAnalysisPrompt } from './combinationalAnalysis'
+import { colorThemes } from './themer/themer'
+import { showTourGuide } from './tutorials'
+import {
+    createVerilogCircuit,
+    // saveVerilogCode,
+    // resetVerilogCode,
+    // applyVerilogTheme,
+} from './Verilog2CV'
+import { generateVerilog } from './verilog'
+import { bitConverterDialog } from './utils'
+import { keyBinder } from '#/components/DialogBox/CustomShortcut.vue'
+import { ExportProject } from '#/components/DialogBox/ExportProject.vue'
+import { ImportProject } from '#/components/DialogBox/ImportProject.vue'
+
+const logixFunction = {
```

Comment:
can we define the type here?

Reply:
this are all mostly imported functions which already have inferred types 
![Untitledcc](https://github.com/user-attachments/assets/ab0e985b-8b5a-4c60-bd14-2b6e03dfc210)